common/lib/capa/capa/templates/codeinput.html

```diff
@@ -1,12 +1,16 @@
+<%page expression_filter="h"/>
```

Comment:
Generally looks good to me.  You just need to make sure any `${x}` will not contain HTML that will now mistakenly be escaped.  For example, if you haven't already, you should confirm that `${response_data['label']}` either never contains HTML or is already a Markup class (which it might be from the other change?).

👍 Once the above is confirmed.